src/Database/Esqueleto/Internal/Language.hs

```diff
@@ -1060,3 +1072,59 @@ instance ( Esqueleto query expr backend
     a <- fromPreprocess
     b <- fromPreprocess
     fromJoin a b
+
+----------------------------------------------------------------------
+-- Functions that Work on lists --------------------------------------
+----------------------------------------------------------------------
+
+-- | @OR@ a list of predicates. An empty list becomes 'False'
+orL :: Esqueleto query expr backend =>
+       [expr (Value Bool)]
+    -> expr (Value Bool)
+orL [] = val False
+orL (p:ps) = List.foldl' (||.) p ps
```

Comment:
This can just be `orL = List.foldl' (||.) (val False)`

Reply:
Yes, they should be identical, but removing the superfluous False results in simpler SQL which makes debugging it easier (especially if you have a bunch of them in a query - which I tend to have).

Reply:
Ah, that makes sense! Might want to leave it as a comment in the code, then :slightly_smiling_face: